src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -163,9 +163,40 @@ function bpFieldInitUploadMultipleElement(element) {
 
 		        fileInput.change(function() {
 	                inputLabel.html("Files selected. After save, they will show up above.");
+					let selectedFiles = [];
+
+					Array.from($(this)[0].files).forEach(file => {
+						selectedFiles.push({name: file.name, type: file.type})
+					});
+
+					element.find('input').first().val(JSON.stringify(selectedFiles)).trigger('change');
 		        	// remove the hidden input, so that the setXAttribute method is no longer triggered
 					$(this).next("input[type=hidden]:not([name='clear_"+fieldName+"[]'])").remove();
 		        });
+
+				element.find('input').on('backpack:field.disable', function(e) {
+					element.children('.backstrap-file').find('input').prop('disabled', 'disabled');
+					element.children('.existing-file').find('.file-preview').each(function(i, el) {
+
+						let $deleteButton = $(el).find('a.file-clear-button');
+
+						if(deleteButton.length > 0) {
+							$deleteButton.on('click.prevent', function(e) {
+								e.stopImmediatePropagation();
+								return false;
+							});
+							// make the event we just registered, the first to be triggered
+							$._data($deleteButton.get(0), "events").click.reverse();
+						}
+					});
+				});
+
+				element.on('backpack:field.enable', function(e) {
```

Comment:
@pxpm can you please put together the PRs to replace the custom event names, from `backpack:field.something` to `CrudField:something`?